kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -662,14 +662,7 @@ def _init_term(
     callvalue: Optional[KInner] = None,
 ) -> KInner:
     program = KEVM.bin_runtime(KApply(f'contract_{contract_name}'))
-    account_cell = KEVM.account_cell(
-        Foundry.address_TEST_CONTRACT(),
-        intToken(0),
-        program,
-        KApply('.Map'),
-        KApply('.Map'),
-        intToken(1),
-    )
+    account_cell = Subst({'TEST_CODE': program})(Foundry.account_TEST_CONTRACT_ADDRESS())
```

Comment:
Hmm, I would 
1. modify `Foundry.account_TEST_CONTRACT_ADDRESS()` to take `program` as a parameter, similar to how `def account_CHEATCODE_ADDRESS(store_var: KInner) -> KApply:` is defined;
2. remove `account_cell` completely;
3. change the `account_cell` from `ACCOUNTS_CELL` on [line 713 ](https://github.com/runtimeverification/evm-semantics/blob/master/kevm-pyk/src/kevm_pyk/foundry.py#L713)with `Foundry.account_TEST_CONTRACT_ADDRESS(program)`.

Reply:
Ah yeah will do, didn't realize nothing else relied on 'TEST_CODE' being there